tasks/install.yml

```diff
@@ -1,26 +1,25 @@
 ---
-- name: detect latest rclone release
```

Comment:
install.yml has many changed lines with no visible changes.
Please check unix/windows line-ending and whitespace settings

Reply:
This problem will be gone after you rebase on master.